src/jukebox/components/gpio/gpioz/plugin/connectivity.py

```diff
@@ -0,0 +1,69 @@
+"""
+Provide connector functions to hook up output devices with some kind of functionality
+"""
+
+import components.volume
+import components.rfid.reader
+import components.gpio.gpioz.plugin
+
+
+def register_rfid_callback(device):
+
+    def rfid_callback(card_id: str, state: int):
+        if state == 0:
+            device.blink(on_time=0.3, n=1)
```

Comment:
I also suggest to use `flash` instead of `blink` btw. It's how we say it in German, but it means something different in English

From [WikiDiff](https://wikidiff.com/blink/flash)

> As verbs the difference between blink and flash is that blink is to close and reopen both eyes quickly while flash is to briefly illuminate a scene.